x/profiles/module.go

```diff
@@ -186,7 +190,9 @@ func (AppModule) ConsensusVersion() uint64 {
 }
 
 // BeginBlock returns the begin blocker for the profiles module.
-func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {
+func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
+	// TODO should we emit an event for this?
+	am.keeper.DeleteExpiredApplicationLinks(ctx)
```

Comment:
should we emit some events for this?

Reply:
Yes, we should emit one event for each deleted link. I don't know whether it would be better to do this here or inside the `DeleteExpiredApplicationLinks` method though

Reply:
I think we can follow how we did with other keeper's methods and put it inside the `DeleteExpiredApplicationLinks` method.